src/azure-cli/azure/cli/command_modules/storage/commands.py

```diff
@@ -259,6 +259,22 @@ def get_custom_sdk(custom_module, client_factory, resource_type=ResourceType.DAT
                                        table_transformer=transform_blob_output,
                                        exception_handler=show_exception_handler)
 
+    blob_lease_client_sdk = CliCommandType(
+        operations_tmpl='azure.multiapi.storagev2.blob._lease#BlobLeaseClient.{}',
+        client_factory=cf_blob_lease_client,
+        resource_type=ResourceType.DATA_STORAGE_BLOB
+    )
+
+    with self.command_group('storage blob lease', blob_lease_client_sdk, resource_type=ResourceType.DATA_STORAGE_BLOB,
```

Comment:
Now we have a new command group "storage blob lease"?

Reply:
discussed offline